src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/Utils/TestMetricReader.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright company="Dynatrace LLC">
+// Copyright 2021 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System.Collections.Generic;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Dynatrace.OpenTelemetry.Exporter.Metrics.Tests.Utils
+{
+	/// <summary>
+	/// A <see cref="MetricReader"/> to be used for tests. It holds the exported items and the export result internally.
+	/// Useful to use for test assertion.
+	/// </summary>
+	internal class TestMetricReader : BaseExportingMetricReader
```

Comment:
Hum yes it will just add to the list. It's not a problem but I'll make sure to clear the internal list it after a call to `GetExportedMetrics`. Good find!